CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## Version 6.6.1
+
+* Correctly set Sentry URL when exporting to mozilla-central ([#2782](https://github.com/mozilla-services/screenshots/issues/2782))
+
+This version includes two changes that addressed Talos performance regressions discussed in [https://bugzil.la/1361792](https://bugzil.la/1361792):
+
+* Do not update toolbar button state inside tab 'activated' event handler ([#2800](https://github.com/mozilla-services/screenshots/issues/2800))
```

Comment:
@dannycoates good catch, I missed one commit when pushing to this branch last night :-P